cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -303,17 +303,47 @@ describe('events dashboard', () => {
     cy.updateEvent(eventId, eventData).then((response) => {
       expect(response.body.errors).not.to.exist;
     });
-    cy.logout();
     // newly registered user (without a chapter_users record)
     cy.register();
 
     cy.login(Cypress.env('JWT_TEST_USER'));
     cy.reload();
     cy.updateEvent(eventId, eventData).then(expectToBeRejected);
-    cy.logout();
+
     // banned admin should be rejected
     cy.login(Cypress.env('JWT_BANNED_ADMIN_USER'));
     cy.reload();
     cy.updateEvent(eventId, eventData).then(expectToBeRejected);
+
+    // admin of chapter 1
+    cy.login(Cypress.env('JWT_CHAPTER_1_ADMIN_USER'));
+    cy.reload();
+    cy.updateEvent(eventId, eventData).then((response) => {
+      expect(response.body.errors).not.to.exist;
+    });
```

Comment:
I might be confused, but this is editing event test, and exactly the same lines are already at the start of _editing_ tests https://github.com/freeCodeCamp/chapter/blob/f12e56e351b2fd7b8b5beaf6cdea707d58d42838/cypress/e2e/dashboard/events/events-index.cy.ts#L300-L305

Reply:
sorry, my bad